.github/workflows/build.yml

```diff
@@ -445,7 +402,7 @@ jobs:
           if-no-files-found: error
 
   build-mssqlmigratorutility:
-    name: Build MsSqlMigratorUtility
+    name: Build MSSQL migrator utility
```

Comment:
⛏️  Here's another case where I think it's better to stick with the package name being built/installed. Or, barring that, fully spell out Microsoft SQL Server.

Reply:
There's a few places where "MSSQL" is used and this is traditionally the shorthand and image name, at least in a lot of places.